e2e/tests/functional/search.e2e.spec.js

```diff
@@ -92,6 +92,7 @@ test.describe('Grand Search', () => {
 });
 
 test.describe("Search Tests @unstable", () => {
+    test.use({ failOnConsoleError: true });
```

Comment:
why is this necessary?

Reply:
The issue the PR is addressing was an error being thrown to console, so by enabling this, we can catch this.

Reply:
Oh, the failOnConsoleError should be set to true by default

Reply:
Ah! That makes sense. I'll remove it.